invenio_users_resources/records/jsonschemas/__init__.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 TU Wien.
+#
+# Invenio-Users-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""JSONSchema directory."""
```

Comment:
thinking about it, i actually doubt that we need any JSON schemas... because we're not storing any JSON data, marshmallow (de-)serialization should be enough

Reply:
👍